File: test/transpiler/pheno_pkg/src/cs/fibonacci.cs
using System;
using System.Collections.Generic;
public class Fibonacci_
{
    public static int fibonacci_(int n)
    {
        //- Description:
    //            - Model Name: fibonacci function
    //            - Author: Pierre Martre
    //            - Reference:  to write in package
    //            - Institution: INRA Montpellier
    //            - Abstract: see documentation
        //- inputs:
    //            - name: n
    //                          - datatype : INT
    //                          - description : argument
    //                          - inputtype : variable
        //- outputs:
    //            - name: result
    //                          - datatype : INT
    //                          - description :  fibonacci number 
        int result;
        int b;
        int temp;
        int i;
        result = 0;
        b = 1;
        for (i=0 ; i<n ; i+=1)
        {
            temp = result;
            result = b;
            b = temp + b;
        }
        return result;
    }
}

File: test/transpiler/pheno_pkg/src/f90/fibonacci.f90
MODULE Fibonacci_mod
    IMPLICIT NONE
CONTAINS
    SUBROUTINE fibonacci_(n, &
        result)
        INTEGER, INTENT(OUT) :: result
        INTEGER:: b
        INTEGER:: temp
        INTEGER:: i
        INTEGER, INTENT(IN) :: n
        !- Description:
    !            - Model Name: fibonacci function
    !            - Author: Pierre Martre
    !            - Reference:  to write in package
    !            - Institution: INRA Montpellier
    !            - Abstract: see documentation
        !- inputs:
    !            - name: n
    !                          - description : argument
    !                          - datatype : INT
    !                          - inputtype : variable
        !- outputs:
    !            - name: result
    !                          - description :  fibonacci number 
    !                          - datatype : INT
        result = 0
        b = 1
        DO i = 1 , n, 1
            temp = result
            result = b
            b = temp + b
        END DO
    END SUBROUTINE fibonacci_
END MODULE
